docs/decisions/0013-microsoft-fhir-converter.md

```diff
@@ -0,0 +1,105 @@
+# 12. Initial Supported File Format
+
+Date: 2022-08-19
+
+## Status
+
+Accepted
+
+## Context and Problem Statement
+
+Given that we have decided to design our buildings blocks around FHIR data, we need a solution for converting Hl7v2 and C-CDA to FHIR. Until recently, this was accomplished by using the `$convert-data` endpoint on the managed implementation of the [Microsoft FHIR server](https://github.com/microsoft/fhir-server) in Azure. However, we have begun porting our ingestion pipeline to Google Cloud Platform (GCP) where an implementation of the Microsoft FHIR server is not available. GCP refers to their implementation of a FHIR server as a [FHIR Store](https://cloud.google.com/healthcare-api/docs/reference/rest/v1/projects.locations.datasets.fhirStores) which does not have a `$convert-data` or similar endpoint. The reason for this is that a `$convert-data` endpoint is not specified as part of the [FHIR API](https://hl7.org/fhir/http.html) standard. In order to fully implement our pipeline in GCP and other platforms in the future we need a solution for FHIR conversion that can be used in any cloud environment or with on-prem infrastructure.     
+
+## Decision Drivers
+
+**Robustness** - The convert must be able to reliably and accurately convert data to FHIR. 
+
+**Hl7v2 Support** - The converter must support the conversion of Hl7v2 to FHIR. 
+
+**C-CDA Support** - The converter must support conversion of C-CDA to FHIR.
+
+**Platform Agnostic** The converter should not depend on any functionality specific to an infrastructure provider.
+
+**Ease of Implementation** - Ideally the converter will be easy implement in a variety of locations.
+
+
+## Considered Options
+
+### Microsoft FHIR Converter CLI
+
+The [Microsoft FHIR Converter](https://github.com/microsoft/FHIR-Converter) is the conversion utility behind the `$convert-data` endpoint on the Microsoft FHIR server and also offers a CLI implementation.
+
+Pros:
+- The same conversion tool we are already familiar with
+- Support for both Hl7v2 and C-CDA
```

Comment:
We should also not that it supports "upscaling" FHIR from STU3 to R4, and will likely support R4 to R5 in the future when R5 comes out.

Reply:
Agreed on STU3 to R4. How do we know that it will "likely support R4 to R5". My thought is that this was really just our own speculation, but not something we actually know. We do know that the MSFT converter is actively maintained and that MSFT is committed to expanding further into healthcare. Maybe getting a little pedantic here.